e2e/tests/workspace/workspace-reader-actions.spec.ts

```diff
@@ -33,11 +33,17 @@ describe('Workspace READER actions tests', () => {
     await workspaceCard.verifyWorkspaceCardMenuOptions(WorkspaceAccessLevel.Reader);
 
     const aboutPage = await workspacesPage.openAboutPage(workspaceCard);
+    await aboutPage.waitForLoad();
+
+    const accessLevel1 = await aboutPage.findUserInCollaboratorList(config.readerUserName);
+    expect(accessLevel1).toBe(WorkspaceAccessLevel.Reader);
```

Comment:
Taken from deleted `workspace-actions-share.spec`.

Reply:
Does this test what the previous test did?

Old test: User A shares Workspace B with reader C, and C logs in to Workspace B to see that they have Reader access.

Does this new test show that the user has Reader access in the same workspace in the first test?  I'm not seeing the connection.

Reply:
The old test was flaky. READER test checks for shared workspace actions permission and checks the About page Collaborator List for the right value. Everything is the same EXCEPT `reader C does not know it was user A who shared workspace B`. I think change is okay because we're not losing any test coverage.